api/types/updatable_project_fields.go

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package types
+
+import (
+	"errors"
+	"fmt"
+)
+
+var (
+	// ErrProjectFieldEmpty is returned when the UpdatableProjectFields is empty.
+	ErrProjectFieldEmpty = errors.New("UpdatableProjectFields is empty")
+
+	// ErrNotSupportedMethod is returned when the method is not supported.
+	ErrNotSupportedMethod = errors.New("not supported method for authorization webhook")
+)
+
+// UpdatableProjectFields is a set of fields that use to update a project.
+type UpdatableProjectFields struct {
+	// Name is the name of this project.
+	Name *string `json:"name,omitempty" bson:"name,omitempty"`
+
+	// AuthWebhookURL is the url of the authorization webhook.
+	AuthWebhookURL *string `json:"auth_webhook_url,omitempty" bson:"auth_webhook_url,omitempty"`
+
+	// AuthWebhookMethods is the methods that run the authorization webhook.
+	AuthWebhookMethods *[]string `json:"auth_webhook_methods,omitempty" bson:"auth_webhook_methods,omitempty"`
+}
+
+// Validate validates the UpdatableProjectFields.
+func (i *UpdatableProjectFields) Validate() error {
+	// Check empty UpdatableProjectFields
+	if i.Name == nil && i.AuthWebhookURL == nil && i.AuthWebhookMethods == nil {
+		return fmt.Errorf("%+v: %w", *i, ErrProjectFieldEmpty)
+	}
```

Comment:
It's always empty, so there doesn't seem to be a need to print it out.

```suggestion
	if i.Name == nil && i.AuthWebhookURL == nil && i.AuthWebhookMethods == nil {
		return ErrEmptyProjectFields
	}
```

Reply:
Comments don't mean much, so I think it's better to delete them.